modules/cvtt/image_compress_cvtt.cpp

```diff
@@ -215,12 +208,15 @@ void image_compress_cvtt(Image *p_image, float p_lossy_quality, Image::UsedChann
 	job_queue.job_params.is_signed = is_signed;
 	job_queue.job_params.options = options;
 	job_queue.job_params.bytes_per_pixel = is_hdr ? 6 : 4;
+	cvtt::Kernels::ConfigureBC7EncodingPlanFromQuality(job_queue.job_params.bc7_plan, 0);
```

Comment:
I really think you should tune the quality parameter, preferably based on p_lossy_quality.  In fact, for BC7, it's probably better to leave flags at Default and tune the quality parameter instead.  Leaving it as 0 should probably only be done if you need maximum speed.  The quality increase at the low end of the quality scale is significant, and low values should still be much faster than the previous CVTT build.

Reply:
Can you describe any tuned numbers?

Reply:
5 will get about 90% of the quality improvement that it's possible to get, 10 will get about 95%, 20 will get about 99%.

Reply:
I'll try 5 and revert Faster/Fastest.

Reply:
Patched to 0, and removed faster/fastest.